python/tk_framework_desktopserver/shotgun/api_v2.py

```diff
@@ -224,10 +224,9 @@ def get_actions(self, data):
 
         manager = sgtk.bootstrap.ToolkitManager()
         manager.base_configuration = self.BASE_CONFIG_URI
-
-        pcs = manager.sort_and_filter_configuration_entities(
```

Comment:
Does this mean we can remove `sort_and_filter_configuration_entities` from core? If this was the only call site for that method, I say kill it.